<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoWidget</class>
 <widget class="QWidget" name="VideoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="videoFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>440</width>
       <height>330</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="6" column="0">
        <widget class="QLabel" name="sourceLabel">
         <property name="text">
          <string>FIle</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="capsLabel">
         <property name="text">
          <string>Number of screencaps</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="KIntNumInput" name="capsInput"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="desintationLabel">
         <property name="text">
          <string>Destination directory</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="KUrlRequester" name="destinationRequester"/>
       </item>
       <item row="6" column="1">
        <widget class="KUrlRequester" name="sourceRequester"/>
       </item>
       <item row="1" column="1">
        <widget class="KComboBox" name="sourceComboBox">
         <item>
          <property name="text">
           <string>File</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DVD</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Source</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="KComboBox" name="chapterComboBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>DVD chapter</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>DVD title</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KComboBox" name="titleComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="KPushButton" name="captureButton">
       <property name="text">
        <string>Capture</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
